### DISCUSSION TITLE
Use English as default language, windows

### DISCUSSION THREAD

**Discussion by AMTMourao** [rustdesk/rustdesk#4428] [Q&A] [answered]
How can I change the app language to English in windows? I'm from Portugal and I use all my software in English.
I can't find where to change to English instead of portugueses?

Thank you
Antonio

**Comment by SergeyMy** [comment]
desktop?
![settings](https://github.com/rustdesk/rustdesk/assets/131688106/84eb2780-0b29-4ea3-af56-ff642c3760b3)

**Reply by AMTMourao** [reply]
![image](https://github.com/rustdesk/rustdesk/assets/44397230/b53d3d1c-39c8-4e30-806b-0592749c3e68)
Hi, How to access that menu?

**Comment by SergeyMy** [comment]
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/446caf48-0002-4856-860e-02bfe7763936)
click on dots :):):):)

**Comment by AMTMourao** [comment]
![image](https://github.com/rustdesk/rustdesk/assets/44397230/8e882780-ec40-4ed3-83ef-61e25ff2d067)
Hi SergeyMy for your quick help. I check the 3 dots is not the same menu.
I going to install a Linux distro and tried to check if I get that menu

**Comment by SergeyMy** [comment, marked as answer]
1. you have an old build with sciter
2. quite strange! (below is a slide with the correct assembly, yours is not credible)

3. the current version is based on flutter libs
use
**NIGHTLY BUILD**
https://github.com/rustdesk/rustdesk/releases/tag/nightly
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/731ee68e-0106-4fcc-b4de-48149b93e4e7)

**Reply by AMTMourao** [reply]
Thank SergeyMy it's works, I was on version 1.1.9, I downloaded from the rustdesk.com website
